Content.Client/MassMedia/Ui/NewsWriteBoundUserInterface.cs

```diff
@@ -72,9 +72,10 @@ private void OnShareButtonPressed()
 
             var stringName = _menu.NameInput.Text;
             var name = (stringName.Length <= 25 ? stringName.Trim() : $"{stringName.Trim().Substring(0, 25)}...");
+            var content = (stringContent.Length <= 2048 ? stringContent.Trim() : $"{stringContent.Trim().Substring(0, 2048)}...");
```

Comment:
why define MaxNameLength and MaxContentLength then not use it in the bui
also just call Trim() when assigning stringName/Content